cypress-shared/support/b2b/utils.js

```diff
@@ -10,11 +10,13 @@ export function getCostCenterName(organization, costCenter) {
 
 export function generateEmailId(organization, role) {
   const [basename, domain] = Cypress.env().base.gmail.id.split('@')
+  const timestamp = new Date().getTime()
 
-  return `${basename}+${Cypress.env().workspace.name}${organization.slice(
-    0,
-    3
-  )}${organization.slice(-1)}${role}@${domain}`
+  return `${basename}+${
+    Cypress.env().workspace.name
+  }${timestamp}${organization.slice(0, 3)}${organization.slice(
```

Comment:
Throughout our testcase - This function must provide constant email.
If it provides different email address then it will cause bug in E2E Tests.

In B2B, we are always using an unique email id for organization-request
eg: 
syed+[workspace][orgname][rolename]@bitcot.com

Once multiorganization request feature is enabled then I will make our test more robust
eg: It will check whether email id has any request if it has then delete and start testcase.

Reply:
Agreed.